skins/oasis/modules/OasisController.class.php

```diff
@@ -87,8 +87,6 @@ public static function onMakeGlobalVariablesScript(Array &$vars) {
 	 * at the bottom for performance reasons, but there are some exceptions for engineering reasons).
 	 *
 	 * TODO: make sure JavaScripts can be always loaded on bottom
-	 *
-	 * Note: NS_FILE pages need JS at top because AnyClips relies on jQuery.
```

Comment:
if this is no longer true, let's get rid of the exception for `NS_FILE`

Reply:
git history for this fragment shows that this comment is not relevant since the code related to NS_FILE check was removed in 2012 🙈